packages/builder/src/pages/builder/auth/_layout.svelte

```diff
@@ -14,7 +14,8 @@
     $admin.cloud &&
     !$admin.disableAccountPortal &&
     $admin.accountPortalUrl &&
-    !$admin?.checklist?.sso?.checked
+    !$admin?.checklist?.sso?.checked &&
+    !$admin.checklist.branding
```

Comment:
This makes me nervous. Should it be `$admin?.checklist?.branding` like the line above, or is the line above incorrect?